app/views/merchants/show.html.erb

```diff
@@ -0,0 +1,113 @@
+<div class="col-md-12 text-center error-message">
+<% if !flash[:error].nil? %>
+  <%= flash[:error] %>
+<% end %>
+</div>
+
+<h1 class="text-center"><%= @merchant.user_name%></h1>
+<div class="col-md-12 text-center gray-button-container">
+  <%= link_to "Add a Product", new_product_path, :class => 'btn black-button' %>
+  <%= link_to "View Products", merchant_products_path(@merchant), :class => 'btn black-button' %>
+  <%= link_to "Add Categories", new_category_path, class: "btn black-button" %>
+</div>
+
+<p>
+  <% @products.each do |product| %>
```

Comment:
Looks like none of this logic here is being evaluated to the view… put it in your controller, yo!